### DISCUSSION TITLE
Top Left Tile in Extruder Panel

### DISCUSSION THREAD

**Discussion by SuperDodge** [KlipperScreen/KlipperScreen#1047] [Q&A]
What is supposed to happen if I click on this tile/button?  When I click on it I get an error that Klipper doesn't know how to interpret the command but I don't know what it is even supposed to do.

**Comment by alfrix** [comment]
it calls for a macro to switch extruder, it has been removed a couple of weeks ago if you have only 1 extruder (ffc5ff3c06a125d95fca39d0bdf1a65744ce250a)
